src/etc/one-context.d/loc-10-network

```diff
@@ -29,32 +29,41 @@ if [ -z "${action}" ] ; then
 fi
 
 if [ -z "${NETCFG_TYPE}" ] ; then
-    case "${os_id}" in
-        alpine)
-            NETCFG_TYPE='interfaces'
-            ;;
-        altlinux)
-            NETCFG_TYPE='networkd nm'
-            ;;
-        debian|devuan|ubuntu)
-            NETCFG_TYPE='interfaces netplan nm networkd'
-            ;;
-        fedora|centos|rhel|almalinux|ol|rocky)
-            NETCFG_TYPE='scripts nm networkd'
-            ;;
-        opensuse*|sles|sled)
-            NETCFG_TYPE='scripts'
-            ;;
-        amzn)
-            NETCFG_TYPE='scripts'
-            ;;
-        freebsd)
-            NETCFG_TYPE='bsd'
-            ;;
-        *)
-            NETCFG_TYPE='none'
-            ;;
-    esac
+    for id in ${os_id}; do
+        case "${id}" in
```

Comment:
The code will work with a single break statement after the case finishes

Reply:
Nice catch, fixed.